src/components/Plants/SpeciesByPlotChart.tsx

```diff
@@ -22,45 +23,44 @@ export default function SpeciesByPlotChart(props: Props): JSX.Element {
     if (selectedPlot) {
       setLabels(selectedPlot?.populations?.map((population) => population.species_scientificName));
       setValues(selectedPlot?.populations?.map((population) => population.totalPlants));
+    } else {
+      setLabels([]);
+      setValues([]);
     }
   }, [selectedPlot]);
 
-  const onChangePlot = (newValue: string) => {
-    if (plots) {
-      const plot = plots.find((p) => p.id.toString() === newValue.toString());
-      if (plot) {
-        setSelectedPlot(plot);
-        updatePlotPreferences(plot.id);
-      }
+  const onChangePlot = (plot: any) => {
+    if (plot) {
+      setSelectedPlot(plot as PlantingSitePlot);
+      updatePlotPreferences(plot.id);
+    } else {
+      setSelectedPlot(undefined);
     }
   };
 
   React.useEffect(() => {
-    if (!plots?.length) {
+    if (!zones?.length) {
       setSelectedPlot(undefined);
       return;
     }
-    const plot = plots.find((p) => p.id.toString() === lastPlot?.toString());
-    setSelectedPlot(plot || plots[0]);
-  }, [lastPlot, plots]);
+    /** Enable after we can support preselection in the plot selector
+     * const plots = zones.flatMap((zone) => zone.plots);
+     * const plot = plots.find((p) => p.id.toString() === lastPlot?.toString());
+     * setSelectedPlot(plot || plots[0]);
+     */
+  }, [lastPlot, zones]);
```

Comment:
We can also comment the lastPlot on the dependencies array until we support preselection